scripts/jenkins/cloud/ansible/roles/list_packages/tasks/list-packages.yml

```
@@ -63,7 +70,8 @@
     label: "{{ item_virtual_hosts._ansible_item_label }}"
 
 - set_fact:
-    _merged_dicts_of_data: "{{ _merged_dicts_of_data | default({}) | combine(_result2.results | selectattr('_ansible_item_label', 'equalto', item._ansible_item_label) | map(attribute='parse_xml') | list | combine(item.parse_xml, recursive=True), recursive=True) }}"
+    dicts_empty: {}
+    _merged_dicts_of_data: "{{ dicts_empty | default({}) | combine(_result2.results | selectattr('_ansible_item_label', 'equalto', item._ansible_item_label) | map(attribute='parse_xml') | list | combine(item.parse_xml, recursive=True), recursive=True) }}"
```

Couldn't you get rid of `_dicts_empty`?:

```suggestion
    _merged_dicts_of_data: "{{ {} | combine(_result2.results | selectattr('_ansible_item_label', 'equalto', item._ansible_item_label) | map(attribute='parse_xml') | list | combine(item.parse_xml, recursive=True), recursive=True) }}"
```

---

This is so weird. When I tried that before it didn t work and I had to put `dicts_empty` there... but now it works. :/